src/api/app/views/webui2/shared/user_or_groups_roles/_list.html.haml

```diff
@@ -1,28 +1,27 @@
+- unless User.current.is_nobody?
+  :ruby
+    mail_subject = "#{home_title} - #{project}"
+    mail_subject += "/#{package}" if package
 - records.each do |record|
-  %tr{ id: "#{type}-#{valid_xml_id(record.to_s)}" }
+  %tr
     %td.align-middle
-      = user_with_realname_and_icon(record) if type == 'user'
-      = link_to(record.to_s, controller: 'groups', action: 'show', title: record.to_s) if type == 'group'
+      = image_tag_for(record, size: 20)
+      = link_to(display_name(record), user_or_group_show_path(record))
```

Comment:
Do all groups have icons in production? thinking otherwise it is going to fail..

Reply:
We use an [identicon](https://en.wikipedia.org/wiki/Identicon) which is based on the email address. If the address is not set we [use](https://github.com/openSUSE/open-build-service/blob/master/src/api/app/helpers/webui/webui_helper.rb#L415) a default icon.

Reply:
Ah! it's true! :+1: